assets/terraform-modules/aws/flatcar-linux/kubernetes/variables.tf

```diff
@@ -113,7 +113,6 @@ variable "asset_dir" {
 variable "network_mtu" {
   description = "CNI interface MTU. Use 8981 if using instances types with Jumbo frames."
```

Comment:
If we change the meaning of this variable, from "CNI interface MTU" to "MTU of physical network of the servers", we should change it accordingly. Also value for jumbo frames should be adjusted. Also, how do we support enabling jumbo frames? Perhaps this can be removed from the description?